### PR TITLE
add missing fas & spinner classes

### PR DESCRIPTION
**Pull request by green3g**
# Description
 - Add missing layer control loading indicator classes

# Checklist

 - [x] `grunt lint` produces no error messages

### CODE REVIEW

**Comment by green3g** [comment]
Wait, I misread those docs.

File: viewer/js/gis/dijit/LayerControl/controls/templates/Sublayer.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
                 <span data-dojo-attach-point="labelNode"></span>
                 <span data-dojo-attach-point="iconNode" class="icon-extra"></span>
             <td class="layerControlTableUpdate">
-                <i data-dojo-attach-point="layerUpdateNode" class="fas ${icons.update} fa-spin layerControlUpdateIcon" style="display:none;"></i>
+                <i data-dojo-attach-point="layerUpdateNode" class="fas ${icons.update} fas fa-spinner fa-spin layerControlUpdateIcon" style="display:none;"></i>
```
Comment (tmcgee) [inline viewer/js/gis/dijit/LayerControl/controls/templates/Sublayer.html]: fas class is now listed twice here.

**Comment by tmcgee** [comment]
Isn't the icon class(es) handled [here](https://github.com/cmv/cmv-app/blob/develop/viewer/js/gis/dijit/LayerControl.js#L41)?

**Comment by green3g** [comment]
I'm totally doing this wrong. Yup.

File: viewer/js/gis/dijit/LayerControl/controls/templates/Sublayer.html (hunk 1 above)
Comment (tmcgee) [reply viewer/js/gis/dijit/LayerControl/controls/templates/Sublayer.html]: the original `fas` class that I included should not be here either. It is already included [here](https://github.com/cmv/cmv-app/blob/develop/viewer/js/gis/dijit/LayerControl.js#L41). Maybe fa-spin should be included there as well to make it all configurable?

**Comment by green3g** [comment]
This class stuff in 2 place is way too confusing. We should pick one or the other. Should I just move it all to the layerControl.icons?

**Comment by tmcgee** [comment]
We also have an inconsistency in classes, `layerControlUpdateIcon` in Control.html/SubLayer.html yet it is also included as `layerControlIcon-Update` in layerControl.icons.

So I agree all classes should be in layerControl.icons and verify that `layerControlUpdateIcon` class is not in cmv.css.

**Comment by green3g** [comment]
Should be good to go, right?

**Comment by tmcgee** [comment]
`layerControlUpdateIcon` class can also be removed from Control.html and SubLayer.html, right?

**Comment by tmcgee** [comment]
@roemhildtg I added some class changes for consistency. Look ok?

**Comment by green3g** [comment]
👍

**Pull request merged**